package.json

```diff
@@ -5,8 +5,8 @@
   "private": true,
   "scripts": {
     "start": "npm run dev",
-    "dev": "NODE_ENV=development ./node_modules/.bin/babel-node ./dev-env/dev.js",
-    "build": "NODE_ENV=production ./node_modules/.bin/babel-node ./dev-env/build.js"
+    "dev": "cross-env NODE_ENV=development babel-node ./dev-env/dev.js",
+    "build": "cross-env NODE_ENV=production babel-node ./dev-env/build.js"
```

Comment:
npm scripts automatically add the path to the dependencies/dev dependencies when run... no need to install the node module globally. 

https://docs.npmjs.com/misc/scripts#path

> If you depend on modules that define executable scripts, like test suites, then those executables will be added to the PATH for executing the scripts. So, if your package.json has this:
> 
> ```
> { "name" : "foo"
> , "dependencies" : { "bar" : "0.1.x" }
> , "scripts": { "start" : "bar ./test" } }
```
> then you could run npm start to execute the bar script, which is exported into the node_modules/.bin directory on npm install.
>